frontend/src/queries/nodes/DataTable/DataTable.tsx

```diff
@@ -115,7 +124,13 @@ export function DataTable({ uniqueKey, query, setQuery, context, cachedResults }
     const isReadOnly = setQuery === undefined
 
     const actionsColumnShown = showActions && isEventsQuery(query.source) && columnsInResponse?.includes('*')
-    const columnsInLemonTable = isHogQLQuery(query.source) ? columnsInResponse ?? columnsInQuery : columnsInQuery
+    const columnsInLemonTable =
+        isHogQLQuery(query.source) ||
+        isWebTopSourcesQuery(query.source) ||
+        isWebTopClicksQuery(query.source) ||
+        isWebTopPagesQuery(query.source)
+            ? columnsInResponse ?? columnsInQuery
+            : columnsInQuery
```

Comment:
This file now has some merge conflicts due to the query features changes.

Reply:
Fixed